src/app/Main/Header/Export/Dialog/Header-ExportDialog.component.tsx

```diff
@@ -53,14 +54,14 @@ export function HeaderExportDialogComponent(props: {
     );
 
     return DialogComponent({
-        isOpen: props.isOpen,
+        isOpen: true,
```

Comment:
might be the time for removing the isOpen prop from the DialogComponent too

Reply:
Done. Commit "Remove "isOpen" to control Dialog"